src/main/java/sg/christopher/duke/entities/Deadline.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Wrapper class for deadlines.
+ */
+package sg.christopher.duke.entities;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
```

Comment:
Consider abstracting deadline and event into a TimedTask abstract class. This allows Event to also have synchronous functionalities.